README.md

```diff
@@ -83,6 +83,30 @@ The .app file appears in the dist directory and can be dragged to
 "My Applications". Please note that actual py2app buils do not work yet. This
 is an aliased build, so it will break if you delete your git clone.
 
+### Windows (experimental)
+Assuming you have no previous python installation, either 
+    * use a package manager like chocolatey (http://chocolatey.org/) to install python3
```

Comment:
Needs a space before this line, so it actually renders as a list

Reply:
Should be fixed.

Reply:
Almost, because of the indent, it becomes a code block. But I'll fix that myself after pulling it in (need to fix some structure things in the README anyway)